src/components/floweditor/FlowEditor.tsx

```diff
@@ -260,12 +258,18 @@ export const FlowEditor = (props: FlowEditorProps) => {
     setFlowValidation('');
   };
 
-  const errorMsg = (
+  const errorMsg = () => (
```

Comment:
Is there any specific reason for this to be a function?

Reply:
Yes, by default we are not having any value for flowValidation so to avoid on load call/assignment to this, I have created a function.